apps/meteor/client/views/omnichannel/departments/NewDepartment.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEndpoint, useTranslation } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
+import type { MutableRefObject } from 'react';
+import React, { useEffect } from 'react';
+
+import { useHasLicenseModule } from '../../../../ee/client/hooks/useHasLicenseModule';
+import PageSkeleton from '../../../components/PageSkeleton';
+import EditDepartment from './EditDepartment';
+import UpgradeDepartments from './UpgradeDepartments';
+
+type NewDepartmentProps = {
+	id: string;
+	reload: () => void;
+	refetchRef: MutableRefObject<() => void>;
+};
+
+const NewDepartment = ({ id, reload, refetchRef }: NewDepartmentProps) => {
+	const getDepartments = useEndpoint('GET', '/v1/livechat/department');
+	const hasLicense = useHasLicenseModule('livechat-enterprise');
+	const { data, refetch, isLoading } = useQuery(['getDepartments'], async () => getDepartments());
```

Comment:
Use a URI-like style for query keys:

```suggestion
	const { data, refetch, isLoading } = useQuery(['omnichannel', 'departments', 'new'], async () => getDepartments());
```